.github/workflows/test-minio.yml

```diff
@@ -0,0 +1,33 @@
+on:
+    push:
+        branches:
+            - master
+            - main
+        paths:
+            - 'minio/**'
+    pull_request:
+        paths:
+            - 'minio/**'
+name: "Tests Minio"
+jobs:
+    Tests:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                go-version:
+                    - 1.19.x
+                    - 1.20.x
+                    - 1.21.x
+        steps:
+            -   name: Install MinIO
+                run: |
+                    run -d --restart always -p 9000:9000 --name storage-minio -e MINIO_ROOT_USER='minio-user' -e MINIO_ROOT_PASSWORD='minio-password' minio/minio server /data
```

Comment:
The keyword "docker" is missing here before "run"

Reply:
done